CDCgov/prime-reportstream
# 10493: create validate schemas command

This PR adds the CLI command to validate the schemas in a particular azure blob container

Test Steps:
1. Recompile the prime CLI tool
2. Run `./prime validateSchemas --schema-type="FHIR" --blob-store-connect="DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1;QueueEndpoint=http://localhost:10001/devstoreaccount1;" --blob-store-container="metadata"`
3. Verify that the output is `No schemas were found, please make sure that you have the correct configuration`
4. Upload the contents of `/src/test/resources/fhirengine/translation/FHIR_to_FHIR/` using azure storage explorer into the metadata container into the path `fhir_transforms/sender/`
5. Run `./prime validateSchemas --schema-type="FHIR" --blob-store-connect="DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1;QueueEndpoint=http://localhost:10001/devstoreaccount1;" --blob-store-container="metadata"`
6. Verify the output is
```
-----
Results from validating FHIR, schemas were valid

1 schemas were validated
-----

Individual Results:

Path validated: http://azurite:10000/devstoreaccount1/metadata/fhir_transforms/sender/simple-transform.yml
Valid: true
Errored while validating: false
```
7. Follow the same steps for HL7

## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

## Checklist

### Testing

### Process

## Linked Issues
- #10493

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
